src/test/scala/org/broadinstitute/hail/SparkSuite.scala

```diff
@@ -24,7 +24,7 @@ class SparkSuite extends TestNGSuite {
   @BeforeClass
   def startSpark() {
     val master = System.getProperty("hail.master")
-    sc = SparkManager.createSparkContext("Hail.TestNG", Option(master), "local[2]")
+    sc = SparkManager.createSparkContext("Hail.TestNG", Option(master), "local[1]")
```

Comment:
This is for debugging so messages print out sequentially. This should get switched back to 2 before merging with master.